templates/package-json.js

```diff
@@ -7,7 +7,9 @@ module.exports = (answers) => {
 		"keywords": answers.keywords,
 		"homepage": `https://registry.origami.ft.com/components/${answers.name}`,
 		"bugs": {
-			"url": `https://github.com/Financial-Times/${answers.name}/issues`
+			"url": `https://github.com/Financial-Times/${answers.name}/issues`,
+			"email": "${answers.email}",
+			"slack": "financialtimes/${answers.slack}"
```

Comment:
Interesting, when I read the part of the spec for this, I thought it meant we only put the channel name and not the organisation name because it states it is our slack organisation.

My reading of it would make this part `"slack": answers.slack`.

Below is the part of the spec:
>The bugs field **may** contain a “slack” property, indicating the support channel on the FT’s internal Slack workspace.